### PR TITLE
Update Table/index.ts to export components

### PR DESCRIPTION
**Pull request by davidpett**
The TableScrollContainer props and the TableDataRenderer were not being exported from the Table package.

### CODE REVIEW

File: packages/@mantine/core/src/components/Table/index.ts [hunk 1]
```diff
@@ -26,3 +26,10 @@ export type {
   TableTfootProps,
   TableTheadProps,
 } from './Table.components';
+export type { 
+  TableScrollContainerProps,
+  TableScrollContainerStyleNames,
+  TableScrollContainerCssVariables,
+  TableScrollContainerFactory,
+} from './TableScrollContainer';
+export type { TableDataRendererProps } from './TableDataRenderer';
```
Comment (rtivital) [inline packages/@mantine/core/src/components/Table/index.ts:35]: This should not be exported from the package. TableDataRenderer is an internal component.

File: packages/@mantine/core/src/components/Table/index.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export {
   TableThead,
 } from './Table.components';
 export { TableScrollContainer } from './TableScrollContainer';
+export { TableDataRenderer } from './TableDataRenderer';
```
[the remark this answers is not in the document]
Comment (davidpett) [reply packages/@mantine/core/src/components/Table/index.ts]: Updated to remove that, but kept the type export. Does that work for you?

File: packages/@mantine/core/src/components/Table/index.ts (hunk 1 above)
Comment (davidpett) [reply packages/@mantine/core/src/components/Table/index.ts]: Removed

**Comment by rtivital** [comment]
Thanks for showing interest in contributing to Mantine! Unfortunately, this pull request cannot be accepted because of one of the following reasons:

- The proposed change [has not been discussed with the maintainer](https://mantine.dev/contribute/#contributing-workflow) – it does not fit the library in some way
- The proposed change is not wanted in the library or implemented incorrectly
- The PR is outdated or has errors in tests

**Pull request closed without merging**